File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.cc
#include <iostream>
using namespace std;

int arr[5];

int main()
{
	for (int i = 0; i < 5; i++)
	{
		cin >> arr[i];
	}

	int num = 1;

	while (true)
	{
		int res = 0;

		for (int i = 0; i < 5; i++)
		{
			if (num % arr[i] == 0) {
				res++;
			}
		}

		if (res >= 3) break;

		num++;
	}

	cout << num;
}

File: 백준/Bronze/1145. 적어도 대부분의 배수/적어도 대부분의 배수.py
arr = list(map(int, input().split()))

num = min(arr)

while True:
    res = 0

    for i in range(5):
        if num % arr[i] == 0:
            res += 1

    if res >= 3:
        break

    num += 1

print(num)